extensions/configuration/issuers.yaml

```diff
@@ -0,0 +1,187 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: issuers.cert.gardener.cloud
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: issuers.cert.gardener.cloud
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |-
+    header:
+      - name: spec.acme.email
+        source: spec.acme.email
+      - source: 'status.state ? status.state : "UNKNOWN"'
+        highlights:
+          informative:
+          - UNKNOWN
```

Comment:
try to be consistent with indents - while yaml supports both, let's stick with an extra indent for lists